02.windows/04.solving-problems/10.low-level-settings/docs.en.md

```diff
@@ -0,0 +1,95 @@
+---
+title: 'Low-level Settings guide'
+taxonomy:
+    category:
+        - docs
+visible: true
+---
+
+## How to reach the Low-level settings
+ 
+> Changing *Low-level settings* can cause problems with the performance of AdGuard, may break the Internet connection or compromise your security and privacy. You should only open this section if you are sure of what you are doing or our support-team has asked you about it.  
+ 
+To go to *Low-level settings*, click *Settings* —> *General settings* —> *Advanced settings* and scroll down. 
+
+## Low-level settings
+
+`Block Teredo when the WFP driver is selected`
+
+If you enable this feature, AdGuard will block Teredo when the WFP driver is selected. 
+
+`Block TCP Fast Open`
+
+If enabled, AdGuard will block TCP Fast Open in the Edge browser. To apply settings, you need to restart the browser. 
+
+`Exclude app from filtering by entering the full path`
+
+If you want AdGuard not to filter any application, specify the full path to it and the app will be excluded from filtering. Separate different paths by semicolons. 
+
+`Enable AdGuard pop-up notifications`
+
+Enable this feature to see AdGuard pop-up notifications. They do not appear too often and contain only important information.
+
+`Automatically intercept filter subscription URLs`
+
+Enable this feature if you want AdGuard to automatically intercept filter subscription URLs (i.e. abp:subscribe and alike) and to open a custom filter installation dialog.
+ 
+`Use redirect driver mode`
+
+If this option is enabled, AdGuard intercepts all the traffic and redirects it to the local proxy server for further filtering. Otherwise, AdGuard will filter all the traffic on the fly, without redirection. 
```

Comment:
```suggestion
If this option is enabled, AdGuard intercepts all the traffic and redirects it to the local proxy server for further filtering. Otherwise, AdGuard will filter all the traffic on the fly, without redirection.

The difference here is that in this case, the system will consider AdGuard to be the sole application that connects to the Internet (other applications are routed through it). The downside is that it will make the system Firewall less effective. The upside is that this approach works a little bit faster.
```